components/CardListGeneral/CardListGeneral.js

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+import sportLocationsData from "../../lib/data/sportLocationsData";
+import Card from "../Card/Card";
+
+export default function CardListGeneral() {
+  return (
+    <>
+      <StyledUl>
+        {sportLocationsData.map((location) => (
+          <Card key={location.id} location={location} />
+        ))}
+      </StyledUl>
+    </>
```

Comment:
```suggestion
      <StyledUl>
        {sportLocationsData.map((location) => (
          <Card key={location.id} location={location} />
        ))}
      </StyledUl>
```
The fragments are only necessary with parent elements in the return statement :)